src/Razor/test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/RazorCompletionContextTest.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Razor.Completion.Test
+{
+    public class RazorCompletionContextTest
```

Comment:
Not worth having the test IMO since the type is a DTO object. I'd nuke the file 😄